render/topologies.go

```diff
@@ -82,8 +82,19 @@ var ProcessNameRenderer = Map{
 // graph by merging the process graph and the container topology.
 var ContainerRenderer = MakeReduce(
 	Map{
-		MapFunc:  MapProcess2Container,
-		Renderer: ProcessRenderer,
+		MapFunc: MapProcess2Container,
+		Renderer: MakeReduce( // We only want processes in container _or_ processes with network connections
+			FilterUnconnected{
+				Renderer: ProcessRenderer,
+			},
```

Comment:
Indeed.  We filter unconnected process and we filter processes not in containers and merge the result, to get the set of processes either in a container or connected to something.